lib/cldr/export/data/country_codes.rb

```diff
@@ -0,0 +1,28 @@
+module Cldr
+  module Export
+    module Data
+      class CountryCodes < Base
+        def initialize
+          super(nil)
+          update(:CountryCodes => country_codes)
+        end
+
+        private
+
+        def country_codes 
+          doc.xpath("//codeMappings/*").each_with_object({}) do |node, hash|
```

Comment:
```suggestion
          select("codeMappings/*").each_with_object({}) do |node, hash|
```

We seem to be using both, but might as well use the helper function.

Reply:
I wasn't able to get this working locally. I tried using both: 

- ```select("codeMappings/*")``` 
- ```select("//codeMappings/*")``` 

Neither ended up working, so I left it as ```doc.xpath("//codeMappings/*")``` for the time being. For the former I was simply getting no output, and the latter threw the exception:

```ERROR: Invalid expression: //ldml///codeMappings/* (Nokogiri::XML::XPath::SyntaxError)```

Reply:
oops yeah you are right does not works for me locally either!